src/liballoc/collections/btree/map.rs

```diff
@@ -493,7 +493,7 @@ impl<K: Ord, V> BTreeMap<K, V> {
         BTreeMap { root: node::Root::shared_empty_root(), length: 0 }
     }
 
-    /// Clears the map, removing all values.
+    /// Clears the map, removing all key-value pairs.
```

Comment:
This one is not internal, but the other BTreeMap functions always talk about pairs.

Reply:
Not sure where I got that from, there's no plural "pairs" at all. That makes it all the funnier that BTreeSet prefers "values" over "elements".